go/vt/vttablet/tabletserver/gc/tablegc.go

```diff
@@ -238,73 +222,81 @@ func (collector *TableGC) Close() {
 // Operate is the main entry point for the table garbage collector operation and logic.
 func (collector *TableGC) Operate(ctx context.Context) {
 
-	addTicker := func(d time.Duration) *timer.SuspendableTicker {
-		collector.initMutex.Lock()
-		defer collector.initMutex.Unlock()
+	dropTablesChan := make(chan string)
+	purgeRequestsChan := make(chan bool)
+	transitionRequestsChan := make(chan *transitionRequest)
 
-		t := timer.NewSuspendableTicker(d, true)
-		collector.tickers = append(collector.tickers, t)
+	tickers := [](*timer.SuspendableTicker){}
+	addTicker := func(d time.Duration) *timer.SuspendableTicker {
+		t := timer.NewSuspendableTicker(d, false)
```

Comment:
The test failures are definitely because of the changes in this PR. They work on `main`, and from what I can tell they are coming from this specific change, of the ticker not being initially suspended. This causes the `tableCheckTicker` to tick, leading to a run of `collector.checkTables` which executes `sqlShowVtTables`. These show up as extra outputs in the vtexplain tests - `      + 	0 ks_unsharded/-: show full tables like '\_vt\_%'

Reply:
I can confirm, if I change that line to again suspend the ticker initially, then all the tests in `vtexplain` work.

Reply:
`vtexplain` is running a vttablet tabletserver with a call like `tsv := tabletserver.NewTabletServer(topoproto.TabletAliasString(t.Alias), config, memorytopo.NewServer(""), t.Alias)`
I don't think vtexplain needs the garbage collector to be running, so we can just remove it entirely from vtexplain. 
cc - @harshit-gangal WDYT?

Reply:
> so we can just remove it entirely from vtexplain.

@GuptaManan100 thank you! The tablet GC is not configurable to just turn off. How would you remove it from vtexplain? Do we need a special flag, only used in testing, to prevent tablet GC from running?

Reply:
I could add this as a `tabletenv.Config` variable?

Reply:
OK, just did that in https://github.com/vitessio/vitess/pull/11972/commits/398d022cc97b41d6f48b61fc0777c8579c144fba, this consistent with `EnableOnlineDDL` and `EnableTxThrottler` etc.  The tests are now good!

Reply:
Yes this is exactly what I had in mind, something like `EnableOnlineDDL`